api/v1/intrusiondetection_types.go

```diff
@@ -27,8 +27,42 @@ type IntrusionDetectionSpec struct {
 	// Only DeepPacketInspection is supported for this spec.
 	// +optional
 	ComponentResources []IntrusionDetectionComponentResource `json:"componentResources,omitempty"`
+
+	// AnomalyDetectionSpec provides configuration for running AnomalyDetectionSpec Component within
+	// IntrusionDetection
+	// +optional
+	AnomalyDetectionSpec AnomalyDetectionSpec `json:"anomalyDetection,omitempty"`
+}
+
+type AnomalyDetectionSpec struct {
+	// StorageType Sets the type of storage to use for storing Anomaly Detection Models. By default it will use the ephemeral
```

Comment:
Sets -> sets

> AD

I think it is best to fully write out abbreviations in our specs, because the CRD comments are part of the user experience.